Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -69,6 +69,8 @@ by using the following commands:
 .. code:: bash
 
       kubectl -n kube-system delete ds kube-proxy
+      # Delete the configmap as well to avoid kube-proxy being reinstalled during a kubeadm upgrade (works only for K8s 1.19 and newer)
```

Comment:
You want to put this below `      kubeadm init --skip-phases=addon/kube-proxy`, as this section is for <=1.15.

Reply:
Don't think it is necessary to delete the configmap as it is not installed if you run kubeadm init.
You only need to delete it if you initialized the cluster with kube-proxy.